src/App.js

```diff
@@ -2,12 +2,55 @@ import React from "react";
 import "./App.css";
 
 class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      count: 0,
+      increment: 1
+    };
+  }
+
+  //event handler to increase counter
+  countClick = () => {
+    this.setState({ count: this.state.count + this.state.increment });
+  };
+
+  alertMessage = () => {
+    alert("You can't afford that!")
+  }
+
+  //event handler to increase increment
+  incrementClick = () => {
```

Comment:
you can make these changes in one setState:

` this.setState({ count: this.state.count - 10,  increment: this.state.increment + 1})`